### PR TITLE
Update role_scout to unstick himself

### PR DESCRIPTION
**Pull request by JetBalsa**
Noticed in tight spots he would get stuck in traffic, or between objects, this fixed him up a little

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
[democratic collaboration](https://github.com/TooAngel/democratic-collaboration)
Approved reviews will speed up the merge, request changes will slow it down.

Please review the PR to help.

File: src/role_scout.js [hunk 1]
```diff
@@ -112,7 +112,12 @@ roles.scout.execute = function(creep) {
       // }
       return false;
     }
-
+    if (creep.isStuck()) {
+      creep.moveRandom();
+      creep.say('ImStuck', true);
+      creep.log(JSON.stringify(creep.memory.last) + ' ' + JSON.stringify(creep.isStuck()));
```
Comment (TooAngel) [inline src/role_scout.js]: If you want to add logging, it should be more descriptive, to find it in the code if you see it.

But I would remove it.

File: src/role_scout.js [hunk 2]
```diff
@@ -112,7 +112,12 @@ roles.scout.execute = function(creep) {
...
+      return false;
```
Comment (TooAngel) [inline src/role_scout.js]: `return true`
This method handles the creep so `true`. Otherwise other methods would be called and interfere with your logic.

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 176/349
    Coefficient: 0.504297994269
    Merging in 4 days 9 hours
    Age 3 days 0 hours

**Pull request merged**